### PR TITLE
Feat/add network config

### PR DESCRIPTION
**Pull request by Karenlrx**
- remove pier message.
- modify message type to pb.message.
- refactor PeerMgr extend from BasicPeermanager.
- add network.toml and networkConfig.

### CODE REVIEW

File: internal/repo/network.go [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+		networkConfig  = &NetworkConfig{}
+		checkReaptAddr = make(map[string]string)
+	)
+
+	id, err := peer2.IDFromPrivateKey(privateKey)
+	if err != nil {
+		return nil, err
+	}
+	id2str := id.String()
+	fmt.Println(id2str)
```
Comment (jzhe886) [inline internal/repo/network.go:39]: remove print

File: internal/repo/network_test.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package repo
+
+import (
+	"fmt"
+	crypto2 "github.com/libp2p/go-libp2p-core/crypto"
+	"github.com/meshplus/bitxhub-kit/crypto"
+	"github.com/meshplus/bitxhub-kit/crypto/asym/ecdsa"
+	"github.com/stretchr/testify/require"
+	"testing"
```
Comment (jzhe886) [inline internal/repo/network_test.go:9]: sort imports

File: internal/repo/network.go (hunk 1 above)
Comment (Karenlrx) [reply internal/repo/network.go:39]: Solved

File: internal/repo/network_test.go (hunk 2 above)
Comment (Karenlrx) [reply internal/repo/network_test.go]: Solved

**Review by jzhe886 — APPROVED** [review]

File: internal/rulemgr/rule_mgr.go [hunk 3]
```diff
@@ -1,12 +1,12 @@
 package rulemgr
 
 import (
+	"github.com/meshplus/bitxhub-model/pb"
```
Comment (DiyiWi) [inline internal/rulemgr/rule_mgr.go:4]: imports sort

**Review by DiyiWi — DISMISSED** [review]

File: internal/rulemgr/rule_mgr.go (hunk 3 above)
Comment (Karenlrx) [reply internal/rulemgr/rule_mgr.go]: Solved

**Review by DiyiWi — APPROVED** [review]

**Review by sandyzhou — APPROVED** [review]

**Pull request merged**